src/AutoRest.CSharp/Common/Output/Models/RestClientBuilder.cs

```diff
@@ -874,6 +893,16 @@ public void AddRequestContext()
                 _parameters.Add(KnownParameters.RequestContext);
             }
 
+            private void AddContentTypeRequestParameter(RequestParameter requestParameter, ICollection<string> requestMediaTypes)
+            {
+                var name = requestParameter.CSharpName();
+                var description = CreateDescriptionWithMediaTypes(requestParameter, requestMediaTypes);
```

Comment:
Let's move `CreateDescriptionWithMediaTypes` next to `AddContentTypeRequestParameter`, make it private and rename into `CreateContentTypeRequestParameterDescription`